UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -1,19 +1,160 @@
-//
-//  ViewController.swift
-//  UIKitDZ
-//
-//  Created by Қадыр Маратұлы on 08.02.2024.
-//
+// ViewController.swift
+// Copyright © RoadMap. All rights reserved.
 
 import UIKit
 
+/// ViewController
 class ViewController: UIViewController {
+    lazy var cakeImage: UIImageView = {
```

Comment:
```suggestion
    private lazy var cakeImage: UIImageView = {
```
ниже так же должно быть, все не используемые за пределами класса свойства должны быть приватными